projects/ion/src/lib/popconfirm/popconfirm.directive.spec.ts

```diff
@@ -214,4 +214,12 @@ describe('Popconfirm disabled host component', () => {
       expect(screen.queryAllByText(confirmText)).toHaveLength(0);
     });
   });
+
+  it('should return false if element is disabled', () => {
+    const element = document.createElement('ion-button');
+    element.setAttribute('ng-reflect-disabled', 'true');
+    const isEnable = directive.elementIsEnabled(element);
+    expect(isEnable).toBe(false);
+    expect(result).toBe(false);
```

Comment:
```suggestion
```

Reply:
tanks. I hadn't seen that you duplicated the variable[😂](https://emojiterra.com/pt/lagrimas-de-alegria/)